packages/client/tests/functional/max_bind_value/tests.ts

```diff
@@ -0,0 +1,75 @@
+import testMatrix from './_matrix'
+// @ts-ignore
+import type { Post, PrismaClient } from './node_modules/@prisma/client'
+
+declare let prisma: PrismaClient
+
+testMatrix.setupTestSuite(() => {
+  const posts: Pick<Post, 'id'>[] = []
+  const numberOfUsers = 20 // TODO: update after release
```

Comment:
This will be bumped to 
```suggestion
  const numberOfUsers = 200
```
shortly after this PR will be merged.